packages/gestalt/src/ErrorMessage.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import Box from './Box.js';
+import formatErrorId from './formatErrorId.js';
+import Text from './Text.js';
+
+type Props = {|
+  errorMessage?: string,
+  id: string,
+|};
+
+export default function ErrorMessage({ errorMessage, id }: Props) {
+  return errorMessage ? (
+    <Box marginTop={1}>
+      <Text color="orange">
+        <span id={formatErrorId(id)}>{errorMessage}</span>
+      </Text>
+    </Box>
+  ) : null;
+}
+
+ErrorMessage.propTypes = {
+  errorMessage: PropTypes.string,
```

Comment:
No need to duplicate the name here - `text`, `message` would be good. Also just using `children` is a good way of not duplicating the name.